x-pack/qa/ml-native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/ScheduledEventsIT.java

```diff
@@ -257,6 +259,81 @@ public void testOnlineUpdate() throws Exception {
         assertEquals(0, buckets.get(8).getScheduledEvents().size());
     }
 
+    /**
+     * An open job that later gets added to a calendar, should take the scheduled events into account
+     */
+    public void testUpdatingOpenedJob() throws Exception {
```

Comment:
We now have 2 tests that deal with online updates. The old one is testing online updates with regard to adding events to a calendar. This one tests online updates with regard to adding the job to a group for which there's calendars. How about we rename those tests to reflect that? `testOnlineUpdate` -> `testAddEventsToOpenJob` and `testUpdatingOpenedJob` -> `testAddOpenedJobToGroupWithCaldendar` are some suggestive names but if you have better ideas go for it.

Reply:
The two hardest problems in software development:

* Cache invalidation
* Naming things

:D